plenum/server/models.py

```diff
@@ -12,29 +13,29 @@
 
 class TrackedMsgs(dict):
 
-    def getKey(self, msg):
+    def _get_key(self, msg):
```

Comment:
I think this is a private API and TrackedMsgs should be abstract class. But it can't because TrackedMsgs is inherited from dict. All TrackedMsgs' methods are overridden in classes descendant with other names. And any one method of TrackedMsgs is not called from outside. If we make its methods public, Prepares, Commits, and InstanceChanges will have a lot of duplicate methods.

Reply:
Leaved methods private after discussion.